workflow-templates/veracode-scan.yml

```diff
@@ -0,0 +1,34 @@
+name: veracode-scan
+on:
+  push:
+    branches: [$default-branch]
+  pull_request:
+    branches: [$default-branch]
+
+jobs:
+  scan:
+    environment: veracode-env
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+      - run: npm run security-pack
+      - name: set tgz name
+        id: vars
+        run: echo "::set-output name=file_to_upload::$(ls | grep tgz)"
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '8'
+      - uses: veracode/veracode-uploadandscan-action@master
+        with:
+          appname: 'NativeChat'
```

Comment:
In an effort to be a bit more generic you can use `${{github.repository_owner}}` here. Not a merge-stopper though

Reply:
I prefer to stay as it is because this is the name of the Veracode application, not the organization name.